src/envShared.ts

```diff
@@ -7,3 +7,4 @@
 export const TEST_BUILD = Boolean(process.env.TEST_BUILD);
 export const IS_PRODUCTION = Boolean(process.env.NODE_ENV === 'production');
 export const IS_DEVELOPMENT = Boolean(process.env.NODE_ENV === 'development');
+export const HASH_SALT = process.env.HASH_SALT || 'defaultSalt';
```

Comment:
Using this variable in Playwirght, so I needed to move it outside t3-env (commonjs vs esm issue)